systemtest/src/test/java/io/strimzi/systemtest/metrics/JmxIsolatedST.java

```diff
@@ -101,6 +118,12 @@ void testKafkaZookeeperAndKafkaConnectWithJMX(ExtensionContext extensionContext)
         assertThat("Result from Kafka JMX doesn't contain right version of Kafka, result: " + kafkaResults, kafkaResults, containsString("version = " + Environment.ST_KAFKA_VERSION));
         assertThat("Result from KafkaConnect JMX doesn't contain right version of Kafka, result: " + kafkaConnectResults, kafkaConnectResults, containsString("version = " + Environment.ST_KAFKA_VERSION));
 
+        // Check Jmx Trans log result
+        String jmxTransPodName = kubeClient().listPodsByPrefixInName(jmxTransName).get(0).getMetadata().getName();
+        String jmxTransResult = kubeClient().logs(jmxTransPodName);
+        String expectedJmxTransContainString = "typeName=type=" + jmxTransMetricTypeName + ",name=" + jmxTransMetricName;
+        assertThat("Result from Kafka JmxTrans doesn't contain correct logs, result: " + jmxTransResult, jmxTransResult, containsString(expectedJmxTransContainString));
```

Comment:
Should we also assert it for some expected value? And not just for the metric name?